### PR TITLE
Comments on dev

### PR DESCRIPTION
**Pull request by matthiasgeihs**

### CODE REVIEW

File: src/types.rs [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+/// Can be calculated with [calc_funding_id].
+pub type FundingId = Hash;
+/// Native balance of the protocol.
+///
+/// Holds balances for multiple assets.
+#[derive(Clone, Default, Debug, PartialEq, JsonSchema)]
+pub struct NativeBalance(cw0::NativeBalance);
+
+#[derive(Serialize, Deserialize, Clone, Default, Debug, PartialEq, JsonSchema)]
+struct EncodedBalance(Vec<(String, [u8; 16])>);
```
Comment (matthiasgeihs) [inline src/types.rs]:
```suggestion
struct EncodableNativeBalance(Vec<(String, [u8; 16])>);
```

Comment (matthiasgeihs) [reply src/types.rs]: Add field names `denom` and `amount`.

Comment (ggwpez) [reply src/types.rs]: Since this is a tuple, i cant just add names. But i could add a type, maybe `EncodableCoin` or something like that?

Comment (matthiasgeihs) [reply src/types.rs]: Yes, I think that would be nice for usage in the backend. Otherwise we are dealing with `Field0` and `Field1`.

**Comment by matthiasgeihs** [comment]
Superseeded by #3

**Pull request closed without merging**